image.yaml

```diff
@@ -23,6 +23,9 @@ envs:
     - name: "KIE_CONTAINER_REDIRECT_ENABLED"
       example: "true"
       description: "Whether KIE Container alias ids will be redirected to generated deployment ids. (optional)"
+    - name: "KIE_JACKSON_DESERIALIZATION_WHITELIST"
+      example: "foo.bar.package,bar.foo.package"
+      description: "To protect EAP 6.4 from jackson databind deserialization vulnerabilities (CVE-2017-15095, CVE-2017-17485, CVE-2018-7489) it is possible to configure a whitelist of classes that can be serialised/deserialised. (optional)"
```

Comment:
Just my 2c but I would prefer to say "recommended" instead of "optional", since the default behavior is unsecured, wdyt?

Reply:
@pszubiak

Reply:
Yes I agree we should emphasize that it's recommended to set this value.